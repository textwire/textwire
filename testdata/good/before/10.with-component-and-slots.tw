<body>
@component('components/with-slots')
@slot<h2>Subtitle</h2>@end
@slot('head')<header>Header</header>@end
@slot('foot')<footer>Footer</footer>@end
@end
</body>