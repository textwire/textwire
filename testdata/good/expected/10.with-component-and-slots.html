<body>
<section>
<header>Header</header>

<h1>This is the title</h1>
<h2>Subtitle</h2>
<p>Some paragraph</p>
<img src="https://via.placeholder.com/150" alt="Placeholder image">

<footer>Footer</footer>
</section>
</body>